nest/nest.py

```diff
@@ -504,6 +508,261 @@ def away_temperature(self, value):
         self._set('device', data)
 
 
+class ProtectDevice(NestBase):
+    @property
+    def _device(self):
+        return self._nest_api._status['topaz'][self._serial]
+
+    @property
+    def _topaz(self):
+        return self._nest_api._status['topaz'][self._serial]
```

Comment:
I think we can either axe the _topaz or just have it return self._device since its the same.